SU-SWS/stanford_profile_helper
# D8CORE-2533: external link adjustments

# READY FOR REVIEW/NOT READY
- (Edit the above to reflect status)

# Summary
- Adjustments for all the different external link styling

# Needed By (Date)
Soon

# Urgency
Not critical

# Steps to Test

1. Check out this branch
1. Clear cache
2. Verify that there's an under line on normal links
1. Verify that the arrow on the Big button looks in proportion with the text

![Screen Shot 2020-08-18 at 4 59 27 PM](https://user-images.githubusercontent.com/284440/90576904-4814cc80-e174-11ea-8b5e-f5bd86cf9357.png)


# Affected Projects or Products
- D8Core, Styling for External Link module

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2533
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)
@ishahism 



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
